.github/workflows/test.yml

```
@@ -56,6 +69,6 @@ jobs:
 
       - name: Run tests
         run: |
-          bundle exec rake spec
-          script/integration.sh
+          BUNDLE_GEMFILE=${{ matrix.gemfile }} bundle exec rake spec
+          MATRIX_GEMFILE=${{ matrix.gemfile}} script/integration.sh
```

missing space for uniformity

```suggestion
          MATRIX_GEMFILE=${{ matrix.gemfile }} script/integration.sh
```